waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/CloseableThriftHiveMetastoreIfaceClientFactory.java

```diff
@@ -43,14 +44,14 @@ public CloseableThriftHiveMetastoreIfaceClientFactory(
 
   public CloseableThriftHiveMetastoreIface newInstance(AbstractMetaStore metaStore) {
     String uris = MetaStoreUriNormaliser.normaliseMetaStoreUris(metaStore.getRemoteMetaStoreUris());
-    String name = metaStore.getName().toLowerCase();
+    String name = metaStore.getName().toLowerCase(Locale.ROOT);
```

Comment:
Conversions to upper or lowercase use the platform's default encoding, which may result in improper conversions when used with international characters. The recommended way is:
* String.toUpperCase( Locale l )
* String.toLowerCase( Locale l )